src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -223,6 +224,8 @@ public bool IsEmpty(DocumentId documentId)
                 }
 
                 await SerializeAsync(serializer, project, result.ProjectId, _owner.NonLocalStateName, result.Others).ConfigureAwait(false);
+
+                AnalyzerABTestLogger.LogProjectDiagnostics(project, result);
```

Comment:
Ah , can you add that comment in your function definition.  
yes , that is my understanding that we want to record this only first time vsix is installed. not sure if per document error count will help us with our analysis here though.